### ISSUE TITLE
Remove obvious outliers

### ISSUE THREAD

**Issue by jxu** [davidreke/chess-ratings-backend#9]
From https://www.chessratingcomparison.com/Graphs some data points are bogus, like 17 on lichess vs 1620 on chess.com, or 3100 lichess vs 1658 chess.com. These should be cleaned up to get a nicer looking plot.

**Comment by CUBICinfinity** [comment]
Rather than creating a new issue, I want to suggest something else that can be done instead or in addition. Use something like robust LM, or even LOESS, instead of OLS. You'll get better fit that way.

**Comment by jxu** [comment]
That is true; another option is linear regression with absolute error as loss as it is more robust to outliers, just as the median is versus mean. The project development seems to have stalled though.
